### ISSUE TITLE
Performance Test Example

### ISSUE THREAD

**Issue by arckumari** [agera-edc/DataSpaceConnector#60]
Feature #28 
After #81
Upstream issue https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1020

## Description

Provide performance test examples and documentation using Gatling and the contract negotiation/data transfer scenario as implemented in #81.

Consideration for future performance tests (out of scope for this story): 
- Is the system becoming slower under load?
- Run performance tests on a e.g. nightly schedule or manually, identify trends.
- Future testing scenarios will be added to the MS3 scope documentation by Paul

## Acceptance Criteria
- [x] A first performance test using the tool(s) defined in #81 is available
- [x] A workflow runs on a nightly schedule and also on demand with configurable settings (e.g. number of clients, duration)
- [x] Documentation on how the tool(s) can be used
- [ ] Documentation on best practices when creating performance tests

## Tasks
- [x] Create runner
- [x] Create scheduled workflow
- [x] Create manual workflow
- [x] Documentation

**Comment by cpeeyush** [comment]
Docs were added as part of upstream [PR 126](https://github.com/agera-edc/DataSpaceConnector/pull/126)

**Comment by cpeeyush** [comment]
Upstream PR merged https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/1029
